### PR TITLE
[WiP] Markdownlint + light editing

### PR DESCRIPTION
**Pull request by mojodna**
Markdownlint yells at me when I open these, so I'm simultaneously (re-)reading everything and cleaning up the formatting (and phrasing in some places) as I go.

### CODE REVIEW

File: api-spec/definitions/STAC-collections.fragment.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ components:
         version:
           title: Collection Version
           type: string
-          example: 1
```
Comment (m-mohr) [inline api-spec/definitions/STAC-collections.fragment.yaml]: I was told by a dataset creator that the Major.Minor.Patch is very software-centered and datssets are usually not versions like that. So the single 1 instead 1.0.0 was on purpose.

Comment (mojodna) [reply api-spec/definitions/STAC-collections.fragment.yaml]: Ah, makes sense.

Are they versioned by date? I was thinking of using an example that's obviously a string (even if it's declared so immediately above).

Comment (m-mohr) [reply api-spec/definitions/STAC-collections.fragment.yaml]: As far as I know it depends on the provider, but an incrementing ID or date seems more reasonable than the Major.Minor.Patch.

**Comment by m-mohr** [comment]
I think that should go into the 0.6.0 RC2 release!

**Comment by matthewhanson** [comment]
@mojodna is this no longer a WiP and ready to merge then?

**Comment by mojodna** [comment]
I’m still working on it, but it represents incremental improvement, so it could be merged now with a follow up for the rest.

**Pull request merged**